app/variablesmanager.cpp

```diff
@@ -99,7 +101,12 @@ void VariablesManager::apiRootChanged()
 
 void VariablesManager::authChanged()
 {
-  QgsExpressionContextUtils::setGlobalVariable( QStringLiteral( "mergin_username" ),  mMerginApi->merginUserName() );
+  QgsExpressionContextUtils::setGlobalVariable( QStringLiteral( "mergin_username" ),  mMerginApi->userAuth()->username() );
+}
+
+void VariablesManager::setUserVariables()
+{
+  QgsExpressionContextUtils::setGlobalVariable( QStringLiteral( "mergin_user_email" ),  mMerginApi->userInfo()->email() );
```

Comment:
Why do we have 2 different slots for the 2 variables (username and user email) listening to 2 different signals?